OneCommunityGlobal/HighestGoodNetworkApp
# update infringement spelling

### Description
Fixes bug DASHBOARD COMPONENT(PRIORITY MEDIUM) 1.
RJ: Bluesquare record value incorrect on dashboard on main. Infringement change missing for this item on Main. 
<img width="665" alt="image" src="https://user-images.githubusercontent.com/9314962/213829380-5415c3b4-6a51-4308-b08c-0c928e41ea4f.png">

### Mainly changes explained:
Update all the spelling of infringment to infringement, and two of the file name.
Note: function `Infringments` in PeopleReport.jsx is renamed as `updateInfringements` in development branch. In case the function is called somewhere else I didn't find, I didn't touch its name for the main and just change its spelling from Infringment to Infringement.

### How to test:
check into current branch
do npm install and ... to run this PR locally
go to dashboard→ summary bar
<img width="827" alt="img" src="https://user-images.githubusercontent.com/9314962/213829315-f9f19f2e-b7dd-4318-9e26-6eb2f834a4e3.png">
